### PR TITLE
Update templates

### PR DESCRIPTION
**Pull request by attitude**
Update templates to generate project with new UI look and layout

### CODE REVIEW

File: packages/cli-common/resources/templates/template-project/admin/index.sass [hunk 1]
```diff
@@ -0,0 +1 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap')
```
Comment (jonasNobile) [inline packages/cli-common/resources/templates/template-project/admin/index.sass:1]: Do we use all weights in admin?

Comment (attitude) [reply packages/cli-common/resources/templates/template-project/admin/index.sass:1]: We use some of them, not all, but by reducing we might break styles for the client that might not expect we did not include some of the weights. If we add just some, we need to document it somewhere, I guess.

**Comment by attitude** [comment]
@jonasNobile what's the status on this?

**Comment by JanTvrdik** [comment]
The layout compatiblity fixes have been done in https://github.com/contember/engine/commit/0806c7d5ab0b1bfcc11259dfcd5f51ab57603b51.

What remains is mostly the font. Why cannot the font be include from admin itself and must be included in the project template?

**Comment by JanTvrdik** [comment]
Also note that the font is also needed for the login page, project list and admin panel included in admin server.

**Comment by attitude** [comment]
I suppose, but I was not the one making this decision, that it allows projects to easily provide their corporate font without bringing in the, otherwise unused Contember font too.

It's either this or we should provide some mean to unregister (not include) the Contember font.

**Comment by JanTvrdik** [comment]
> It's either this or we should provide some mean to unregister (not include) the Contember font.

Does browser start dowloading the font even though it's not used?

**Comment by attitude** [comment]
> Does browser start dowloading the font even though it's not used?

Seems it's not used when not referenced.

> A common misconception is that a font is requested when a @font-face declaration is encountered—this is not true. By itself, @font-face declaration does not trigger font download. Rather, a font is downloaded only if it is referenced by styling that is used on the page. For example, like this:

```
@font-face {
  font-family: "Open Sans";
  src: url("/fonts/OpenSans-Regular-webfont.woff2") format("woff2");
}

h1 {
  font-family: "Open Sans"
}
```

Full reference: https://web.dev/font-best-practices/, section **Font loading**.

I'll add the font to the core CSS.

File: packages/cli-common/resources/templates/template-project/admin/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html>
 	<head>
 		<meta charset="utf-8">
-		<meta name="viewport" content="width=device-width">
+		<meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"/>
```
Comment (attitude) [inline packages/cli-common/resources/templates/template-project/admin/index.html:5]: @JanTvrdik any particular reason you omitted this when implementing layout changes in https://github.com/contember/engine/commit/0806c7d5ab0b1bfcc11259dfcd5f51ab57603b51?

Comment (JanTvrdik) [reply packages/cli-common/resources/templates/template-project/admin/index.html:5]: AFAIK `initial-scale=1` is useful only for old Safari. And I don't have phone with a notch to test if `viewport-fit=cover` does anything useful.

**Comment by matej21** [comment]
we can close this PR now, right?

**Pull request closed without merging**